apiserver/facades/client/modelgeneration/interface.go

```diff
@@ -17,6 +18,7 @@ type State interface {
 	ControllerTag() names.ControllerTag
 	Model() (Model, error)
 	Application(string) (Application, error)
+	WriteBulkSettings([]state.Settings) error
```

Comment:
I'm not sure why we need this as part of the State interface in the apiserver.

Reply:
Gone.